### ISSUE TITLE
Environments without DB still require 1GB DB capacity?

### ISSUE THREAD

**Issue by VitalyMCT** [MicrosoftDocs/power-platform#1453]
This may be a duplicate of https://github.com/MicrosoftDocs/power-platform/issues/413.

In the "Create an environment without a database" section, it states:

> Prerequisites: You need 1GB available database capacity.

Is this correct? If so, where does the need for available database capacity come from if there is no new DB created?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 00543757-56fc-9709-49be-16970ffbb774
* Version Independent ID: c2c1da0b-c7a4-1128-f9de-1f2fae684a11
* Content: [Create and manage environments in the Power Platform admin center - Power Platform](https://docs.microsoft.com/en-us/power-platform/admin/create-environment)
* Content Source: [power-platform/admin/create-environment.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/admin/create-environment.md)
* Service: **power-platform**
* Sub-service: **admin**
* GitHub Login: @jimholtz
* Microsoft Alias: **jimholtz**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @VitalyMCT. This issue has been assigned to the doc author for review.

**Comment by jimholtz** [comment]
@VitalyMCT 
1GB is required even for environments created without a database. We don’t want to prevent customers from creating a database after they have created an environment (due to lack of capacity at a later time).
